docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Lack of Unique IDs and Aria-Labels on Sections/Categories, Hindering Automation
+
+There is a lack of unique IDs applied to sections and categories, which hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
+
+.Support disconnected installations
+
+MTA does not support disconnected installations. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Cannot remove the Jira connection to a Wave that is later removed 
+
+It is not possible to remove a Jira connection to a Wave, after that Wave is later removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
+
+.Binary analysis takes 3 hours and then fails
+
+Binary analysis can take 3 hours, and then it fails. link:https://issues.redhat.com/browse/MTA-1183[MTA-1183]
+
+.Allow architects to review an application without an assessment
+
+Allow architects to review an application without an assessment. link:https://issues.redhat.com/browse/MTA-115[MTA-115]
+
+.Dependencies tab shows `error` when retrieving data
+
+An error message appears on the MTA UI when retrieving data.link:https://issues.redhat.com/browse/MTA-1130[MTA-1130]
+
+.No update notification after editing fields
+
+No `Update Notification` appears at top of window after the fields `Application`, `Job Function`, and `Business services` are updated. link:https://issues.redhat.com/browse/MTA-1024[MTA-1024]
+
+.Disabled `rank` field when creating a new tag category
+
+After creating a new tag category, the `rank` field text box is disabled. link:https://issues.redhat.com/browse/MTA-1095[MTA-1095]
+
+.Cannot create a Jira instance behind a proxy
+
+Red Hat Jira instance (issues.stage.redhat.com) is accessible only through a proxy, but general proxy settings do not work with this configuration. link:https://issues.redhat.com/browse/MTA-849[MTA-849]
+
+.Rows in `Dependencies` show placeholder instead of actual value
+
+In the `Migration -> Dependencies` view, some rows show a placeholder under the `version` column instead of the actual value. link:https://issues.redhat.com/browse/MTA-1283[MTA-1283]
+
+.Source Analysis shows different Issues for `tackle-testapp`
+
```

Comment:
Sentence case for headings (see IBM above): "Issues" should not be capitalized.  "Analysis" is questionable in the heading. In the UI itself "Source Analysis" should be "Source analysis" if the phrase actually appears in the UI. Please check the UI and follow it. If there is no "Source analysis" UI element, then write "Source analysis" in the header and "source analysis" in the text, without using bold.